packages/airnode-deployer/src/handlers/aws/index.ts

```diff
@@ -1,9 +1,10 @@
 import * as path from 'path';
 import { handlers, logger, utils, providers, WorkerResponse } from '@api3/airnode-node';
 import { loadConfig } from '../../utils';
+import { isCloudFunction } from '../../utils/infrastructure';
 
 const configFile = path.resolve(`${__dirname}/../../config-data/config.json`);
-const parsedConfig = loadConfig(configFile, process.env);
+const parsedConfig = loadConfig(configFile, process.env, !isCloudFunction());
```

Comment:
Do we even need the `isCloudFunction` function? Neither this, nor the GCP handler are called locally (there's a local handler for that in `airnode-node` package). So we can pass `false` directly.

Reply:
Ahhh, wow, yes, thank you, I'll amend :)